tests/ci/cdk/cdk/components.py

```diff
@@ -1,75 +1,115 @@
 import pathlib
+import typing
 
-from aws_cdk import aws_codebuild as codebuild, aws_lambda as lambda_, aws_ecr as ecr, aws_secretsmanager as sm, \
-    aws_events as events, aws_events_targets as events_targets, aws_iam as iam, Duration
+from aws_cdk import (
+    aws_codebuild as codebuild,
+    aws_lambda as lambda_,
+    aws_ecr_assets as ecr_assets,
+    aws_secretsmanager as sm,
+    aws_events as events,
+    aws_events_targets as events_targets,
+    aws_iam as iam,
+    Duration,
+    Environment,
+)
 
 from constructs import Construct
-from util.metadata import AWS_REGION, AWS_ACCOUNT, GITHUB_REPO_OWNER, GITHUB_TOKEN_SECRET_NAME
+from util.metadata import GITHUB_REPO_OWNER, GITHUB_TOKEN_SECRET_NAME
 
 
 class PruneStaleGitHubBuilds(Construct):
-    def __init__(self, scope: Construct, id: str, *, project: codebuild.IProject, ec2_permissions: bool) -> None:
+    def __init__(
+        self,
+        scope: Construct,
+        id: str,
+        *,
+        project: codebuild.IProject,
+        env: typing.Union[Environment, typing.Dict[str, typing.Any]],
```

Comment:
I put it earlier, and I've glossed over it in a few places, but this should be the same everywhere right?